File: py/2706.py
class Solution:
    def buyChoco(self, prices:List[int], money:int) -> int:
        min1, min2 = 128,128
        for x in prices:
            if min1 >= x:
                min2 = min1
                min1 = x
            elif min2 > x:
                min2 = x
        leftover = money - min1 - min2
        return leftover if leftover >= 0 else money

File: cpp/2706.cpp
class Solution {
public:
  int buyChoco(vector<int>& prices, int money) {
    int min1 = 128, min2 = 128;
    for (int n : prices) {
      if (min1 >= n) {
        min2 = min1;
        min1 = n;
      }
      else if (min2 > n) {
        min2 = n;
      }
    }
    int leftover = money - min1 - min2;
    return leftover >= 0 ? leftover : money;
  }
};
